File: Java/1009.java
import java.io.IOException;
 
import java.util.Locale;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Locale.setDefault(Locale.US);
        Scanner entrada = new Scanner(System.in);
        
        String nome = entrada.next();
        double salarioFixo = entrada.nextDouble();
        double montante = entrada.nextDouble();
        
        double total = (0.15 * montante) + salarioFixo;

        System.out.printf("TOTAL = R$ %.2f%n", total);
        entrada.close();
    }
}

File: Go/1009.go
package main
 
import (
    "fmt"
)
 
func main() {

    var nome string
    var salarioFixo, montante, total float64

    fmt.Scanf("%s", &nome)
    fmt.Scanf("%f", &salarioFixo)
    fmt.Scanf("%f", &montante)

    total = ((montante * 15) / 100) + salarioFixo

    fmt.Printf("TOTAL = R$ %.2f\n", total)
}
